landside/catkin_ws/src/simulation/scripts/squareCommand.py

```diff
@@ -1,19 +1,13 @@
-import thread
+#!/usr/bin/env python
 import time
 
 import rospy
 from std_msgs.msg import Float32MultiArray
 
-
-def input_thread(a_list):
-    input()
-    a_list.append(True)
-
-
 if __name__ == "__main__":
     pub = rospy.Publisher("/sim/move", Float32MultiArray, queue_size=10)
     rospy.init_node("move")
-    start = time.time()
+    start = rospy.get_rostime().secs
```

Comment:
Just set this to rospy.get_rostime(), subtracting to ros times gives you a duration which you can use to_sec() to convert to a floating point seconds.